### ISSUE TITLE
Measure tools widget always loads but is blank.

### ISSUE THREAD

**Issue by deanmarcussen** [iTwin/viewer-components-react#184]
After upgrading to v3 the Measurements widget seems to always autoload, as a widget, but blank with nothing on it.

![image](https://user-images.githubusercontent.com/13782679/153169916-7758ce97-8032-4edf-b104-41640ca04d44.png)

I forced upgraded to the implicit reference to 0.3.0 with no change.

Currently calling this as a workaround when the view is created, as I don't especially want the widget
```
UiItemsManager.unregister("MeasureToolsUiItemsProvider");
```

**Comment by aruniverse** [comment]
Hey Dean, 

The widget is empty until you make an actual measurement and select it, at which point it will display the properties of the measurement. We'll look into only displaying it when a measurement is selected.

**Comment by deanmarcussen** [comment]
Cool, for my case, we prefer not to have the widget, we have too many custom ones already :)

I wasn’t sure if it was supposed to be part of the features set, as the timeline component moved to that arrangement and disappeared for a while until I discovered that.

also the readme says it still needs to be initialized, so maybe needs a tweak?

**Comment by aruniverse** [comment]
You can disable the widget along with the measure-tools by setting the `defaultUiConfig` prop to:

```tsx
const defaultUiConfig: ItwinViewerUi = {
  contentManipulationTools: {
    verticalItems: {
      measureTools: false
    }
  }
};
```

The readme describes how to use the pkg individually. We add it to the itwin-viewer by default and initialize measure-tools in there, so it doesn't exactly apply

**Comment by aruniverse** [comment]
@deanmarcussen , this should be fixed in the v0.7.0 of the measure-tools-react pkg

**Comment by deanmarcussen** [comment]
Thanks I'll try and get the upgrade to all out this week.
